packages/melmatcheq.lv2/PKGBUILD

```diff
@@ -0,0 +1,48 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_name=MelMatchEQ.lv2
+pkgname=${_name,,}
+pkgver=0.1
+pkgrel=1
+pkgdesc='A profiling EQ using a 26 step Mel Frequency Band LV2 plugin'
+arch=(aarch64 x86_64)
+url="https://github.com/brummer10/$_name"
+license=(GPL2)
```

Comment:
Though the source dist contains a `LICENSE` file with the GPL3 text, GPL2 was intended:

https://github.com/brummer10/MelMatchEQ.lv2/issues/1